README.md

```diff
@@ -1 +1,73 @@
-# Mobile-version
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```

Comment:
- [ ] Please the main container of the projects  should have a display:grid since the grid is used for tow dimensional aligning while flexbox is for one-dimensional aligning

- [ ]  The section should have  the proper background image (decorations are always backgrounded images, not img tags)

| Your design | Figma design |
| --------------- | ------------------- |
| ![image](https://user-images.githubusercontent.com/74744787/176790371-fe2b3102-06ef-4ca1-9b8c-c063ebd4f5bb.png) | ![image](https://user-images.githubusercontent.com/74744787/176790300-2227aab1-9366-44b1-8778-2718fe29d39b.png) |

- [ ] Please For each project The content of the project should be  aligned with the flexbox

- [ ] Please The button (e.g., "See project") should be a link a or button tag, and follow the button styling: Active, Hover, and Pressed (see design)

- [ ] Please The section should have the margins (top, bottom, left, right) defined in the design

Reply:
All done, l will push each file with the propper commit.

Thank you